react/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "redux-loop/react",
+  "description": "React specific declarations for redux-loop",
+  "homepage": "https://github.com/redux-loop/redux-loop",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/redux-loop/redux-loop.git"
+  },
+  "bugs": {
+    "url": "https://github.com/redux-loop/redux-loop/issues"
+  },
+  "files": [
```

Comment:
we should actually make sure to include the whole react-redux folder in the files array in the main package.json. not this one.